index.html

```diff
@@ -1,6 +1,16 @@
 <html>
+    <link rel="stylesheet" href="styles.css"> 
     <head>
-
+                <nav>
+            <div id="logo">
+                <img src="img/youtube-logo.png" alt="Youtube Company logo">
+            </div>
+            <input type="text" placeholder="Search your music"></input>
+            <button type="submit"></button>
+            <img src="img/notification-icon.png" alt="Notification Icon">
+            <img src="img/upload-icon.png" alt="Upload Icon">
+            <img src="img/avatar-icon.png" alt="Avatar Icon">
+        </nav>
```

Comment:
The code above including the images, div, input and button should be in the body tag and not the head tag.